sdcm/tester.py

```diff
@@ -762,6 +757,33 @@ def clean_resources(self):
         if self.monitors is not None:
             self.monitors.get_backtraces()
             self.monitors.download_monitor_data()
+
+            # upload prometheus data
+            prometheus_folder = glob.glob(os.path.join(self.logdir, '*monitor*'))[0]
+            file_path = os.path.join(self.logdir, os.path.basename(os.path.normpath(self.job.logdir)))
```

Comment:
You create the zip file insider self.logdir, how about compress monitor log to the top of self.logdir?

file_path = os.path.normpath(self.job.logdir)